packages/assets-controllers/src/NftController.ts

```diff
@@ -291,6 +294,8 @@ export class NftController extends BaseController<
     source: Source;
   }) => void;
 
+  #lastNftRefreshTime = 0;
```

Comment:
What's the thinking behind doing this vs having the server cache with `max-age`?  The server cache seemed convenient since it'd automatically be per-url.  vs having to know when to reset this.

Reply:
Good point on the cache! Unlike extension where im able to verify that the request didnt actually hit the server and i got the response from disk cache, on mobile i think i didnt have the proper setup, but i think its the same.
I think adding the mutex solved the big part of this issue, and i can remove the lastNftRefreshTime property.